DOCS-12316-httpsPortsPRfromNathan-v44-duke (#2121)
mulesoft/docs-mule-runtime: modules/ROOT/pages/build-app-practices.adoc

----

=== HTTP and HTTPS

> HTTP and HTTPS property placeholders are reserved in CloudHub deployments. Use of the same placeholders is recommended for on-premises deployments.

Author: ```suggestion
Property placeholders are reserved for HTTP and HTTPS ports in CloudHub deployments, and the defaults for these properties are used by the shared load balancer in CloudHub. Use of the same placeholders is recommended for on-premises deployments.
```

> Property placeholders are reserved for HTTP and HTTPS ports in CloudHub deployments, and the defaults for these properties are used by the shared load balancer in CloudHub. Use of the same placeholders is recommended for on-premises deployments.

Author: ```suggestion
Property placeholders are reserved for HTTP and HTTPS ports in CloudHub deployments. The shared load balancer in CloudHub uses the defaults for these properties. 

Use of the same placeholders is recommended for on-premises deployments.
```

> * In CloudHub, `${http.port}` maps to the setting `port=8081`, and `${https.port}` maps to `port=8082`.

Reviewer: We should mention that those ports are the default used by the shared load balancer in CH. Not sure if there is any doc in CH that explains that

Author: ```suggestion
* In CloudHub, `${http.port}` maps to the setting `port=8081`, and `${https.port}` maps to `port=8082`. The ports are the defaults used by the shared load balancer in CloudHub. 
```

Author: Revised the initial paragraph to include info on the load balancers. There's a link to the CH info in "For details, ..." at the end of this section.

Author: @facunlk can you check the changes and approve if warranted. Same changes are also in https://github.com/mulesoft/docs-mule-runtime/pull/2120/files. Thx!

> * For internal ports to HTTP services in CloudHub deployments, `${http.private.port}` maps to the setting `port=8091`, and `${https.private.port}` maps to `port=8092`.

Reviewer: Similar to the previous item. Those internap ports are only relevant because it is the default port used by the dedicated load balancers. Can we clarify that or reference another doc with that information?

Author: ```suggestion
* For internal ports to HTTP services in CloudHub deployments, `${http.private.port}` maps to the setting `port=8091`, and `${https.private.port}` maps to `port=8092`. The ports are the defaults used by the shared load balancer in CloudHub. 
```

Author: see above

----

=== HTTP and HTTPS

> For HTTP and HTTPS ports in on-premises deployments, MuleSoft recommends using the names for property placeholders that are reserved for CloudHub deployments. 

Reviewer 2: MuleSoft recommends using > use

Author: I don't think we should suggest that they must use those names. I know it breaks our writing style protocol, but i think our protocol is incorrect in this case.

----

=== HTTP and HTTPS

> Property placeholders are reserved for HTTP and HTTPS ports in CloudHub deployments. The shared load balancer in CloudHub uses the defaults for these properties. 

Author: ```suggestion
For HTTP and HTTPS ports, MuleSoft recommends using the names for property placeholders that are reserved for CloudHub deployments. 
The shared load balancer in CloudHub uses the defaults for these properties. 
```

----

=== HTTP and HTTPS

> For HTTP and HTTPS ports, MuleSoft recommends using the names for property placeholders that are reserved for CloudHub deployments. 

Author: ```suggestion
For HTTP and HTTPS ports in on-premises deployments, MuleSoft recommends using the names for property placeholders that are reserved for CloudHub deployments. 
The shared load balancer in CloudHub uses the defaults for these properties. 
```
